src/library/scala/collection/mutable/ArrayOps.scala

```diff
@@ -202,7 +202,7 @@ object ArrayOps {
   }
 
   /** A class of `ArrayOps` for arrays containing `byte`s. */
-final class ofByte(override val repr: Array[Byte]) extends AnyVal with ArrayOps[Byte] with ArrayLike[Byte, Array[Byte]] {
+  final class ofByte(override val repr: Array[Byte]) extends AnyVal with ArrayOps[Byte] with ArrayLike[Byte, Array[Byte]] {
```

Comment:
In my experience whitespace edits are accepted when accompanied by more significant changes and not otherwise.